backend/cmd/migrate/migrations/000009_experimentation_id_column_type_change.up.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE experiment_run ALTER COLUMN "id" SET DATA TYPE varchar(60) USING id::varchar;
```

Comment:
still veryfing what character limit makes sense here.

Reply:
the longest service name I found was 46 characters long. I think that 100 character limit should be fine.

Reply:
going with 100 characters limit for now.